### ISSUE TITLE
Social Auth Settings placed incorrectly in Portal Config

### ISSUE THREAD

**Issue by rfun** [tethysplatform/tethys#600]
Hi, 

**Task**: I was trying to implement Social Auth using Hydroshare for our portal
**Procedure**: I followed the instructions on http://docs.tethysplatform.org/en/latest/installation/production/configuration/advanced/social_auth.html

**Issue**: When I run the tethys set settings command  it is putting it under the OAUTH_CONFIG section in the portal_config file. However, the hs_restclient_helper is using the following command to get the setting: 

`                client_id = getattr(settings, "SOCIAL_AUTH_{0}_KEY".format(backend_name.upper()), 'None')
`

Basically, there is a mismatch between where the settings command puts in the key and secret vs. where its looking for those settings. 

Manually editing the portal_config to just add those settings outside the OAUTH_CONFIG block and at the main `settings` level, works just fine. 

Let me know if you need more information.

**Comment by swainn** [comment]
@rfun Thanks for reporting. I have a few questions:

- What version of Tethys are you encountering this in? 

- What is the ``tethys set`` command you are running that is causing the issue?

**Comment by rfun** [comment]
This was on tethys 3.0.0. I just checked the latest hs_restclient code in tethys_services and it looks to be the same. 

The set command was the one provided in the documentation: 
`tethys settings --set OAUTH_CONFIG.SOCIAL_AUTH_HYDROSHARE_KEY <Client id> --set OAUTH_CONFIG.SOCIAL_AUTH_HYDROSHARE_SECRET <Client secret>`

So I guess one thing could be is to just update the documentation to use the set command without the OAUTH_CONFIG, or update the settings.py code to read the oauth_config block properly?

**Comment by swainn** [comment]
@rfun you ought to try [updating to 3.1.0](http://docs.tethysplatform.org/en/stable/installation/update.html), which was released recently. We addressed some issues with the settings and portal_config in that release.

**Comment by rfun** [comment]
You are right. It was fixed in the newer version. I tried it in 3.1.1 and it worked according to the docs
